core/drivers/crypto/caam/acipher/caam_rsa.c

```diff
@@ -73,6 +73,10 @@ struct caam_rsa_keypair {
 	struct caambuf qp; /* Private [qp = 1/q mod p] */
 };
 
+#define RSA_PRIVATE_KEY_FORMAT_1 1
```

Comment:
I can see why we didn't have these before, since the name of the key format happens to be a number. It's still an improvement though.

Reply:
It helps with readability at least.
Tag applied!